### PR TITLE
add edexa chain

### PR DESCRIPTION
**Pull request by ranjith-edx**
Testnet:
chainID: 1995
networkID: 1995

### CODE REVIEW

File: _data/icons/edexa.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmeZW6VKUFTbz7PPW8PmDR3ZHa6osYPLBFPnW8T5LSU49csdf",
+    "width": 400,
+    "height": 400,
+    "format": "png"
```
Comment (ligi) [inline _data/icons/edexa.json:6]: ✗ ~/tmp ipfs get QmeZW6VKUFTbz7PPW8PmDR3ZHa6osYPLBFPnW8T5LSU49csdf      19:35:51
Error: invalid path "QmeZW6VKUFTbz7PPW8PmDR3ZHa6osYPLBFPnW8T5LSU49csdf": selected encoding not supported

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/icons/edexa.json (hunk 1 above)
Comment (ranjith-edx) [reply _data/icons/edexa.json]: Hi @ligi  i have runneded with pretier still shows changes required

**Comment by ligi** [comment]
seems you did not run the prettier correctly

File: _data/icons/edexa.json (hunk 1 above)
Comment (ranjith-edx) [reply _data/icons/edexa.json]: commited back with running prettier again

Comment (ranjith-edx) [reply _data/icons/edexa.json]: all checks have been passed it can be merged now right.?! @ligi

**Review by ligi — CHANGES_REQUESTED** [review]
tl;dr: it is now required to put the binary icon into iconDownload

why: it was a PITA to always `ipfs get` in the review process - sometimes it takes *really long* or even fails (especially with pinata) - this took significant time (which adds up with the amount of chains added) and slowed down the review process. Also sometimes CIDs become unavailable after - big mess. Hence it is now required to put them in iconDownload (still named via CID)

**Comment by ligi** [comment]
also the prettier still not happy

File: _data/icons/edexa.json (hunk 1 above)
Comment (ranjith-edx) [reply _data/icons/edexa.json]: need to just add the image in the iconsdownload in cid name without extension right

Comment (ligi) [reply _data/icons/edexa.json]: yes like the others

Comment (ranjith-edx) [reply _data/icons/edexa.json]: running prettier adding new line to the files at the end i think i have fixed it

Comment (ranjith-edx) [reply _data/icons/edexa.json]: hi please review and accept changes @ligi

Comment (ligi) [reply _data/icons/edexa.json]: image

will not review with so many files changed ..

Comment (ranjith-edx) [reply _data/icons/edexa.json]: reverted back the commits and runned the prettier

File: _data/icons/edexa.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmWFMLXD87rPB6KxjooTUc1a6o3P856fT8WKUSA6VhToHV",
+    "width": 400,
+    "height": 400,
+    "format": "png"
```
Comment (ligi) [inline _data/icons/edexa.json:6]: identify QmWFMLXD87rPB6KxjooTUc1a6o3P856fT8WKUSA6VhToHV         14:41:38
QmWFMLXD87rPB6KxjooTUc1a6o3P856fT8WKUSA6VhToHV JPEG 1028x1042 1028x1042+0+0 8-bit sRGB 53936B 0.000u 0:00.000

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/icons/edexa.json (hunk 2 above)
Comment (ranjith-edx) [reply _data/icons/edexa.json:6]: added

Comment (ranjith-edx) [reply _data/icons/edexa.json:6]: > identify QmWFMLXD87rPB6KxjooTUc1a6o3P856fT8WKUSA6VhToHV 14:41:38
> QmWFMLXD87rPB6KxjooTUc1a6o3P856fT8WKUSA6VhToHV JPEG 1028x1042 1028x1042+0+0 8-bit sRGB 53936B 0.000u 0:00.000

it should be in jpg format and size 1028x1042 ..?!

Comment (ranjith-edx) [reply _data/icons/edexa.json:6]: it should be in jpg format and size 1028x1042 ..?! @ligi

Comment (ligi) [reply _data/icons/edexa.json]: the format just has to match the metadata

Comment (ranjith-edx) [reply _data/icons/edexa.json]: i have added the icon variable to the eip155-1995.json file and its all matching  and the format is PNG as it is as well as size are same

Comment (ranjith-edx) [reply _data/icons/edexa.json]: not sure what is the error now is there anything need to be changed..?! @ligi

Comment (ranjith-edx) [reply _data/icons/edexa.json]: Hi @ligi  it is actually a png image and in json it also mentioned as png file still gives this error 

image

Comment (ranjith-edx) [reply _data/icons/edexa.json]: @ligi  both jpg and png gives vise versa errors kindly look on to this 

image

**Review by ligi — CHANGES_REQUESTED** [review]
a jpeg is the worst fileformat for this - please make it either a PNG or a SVG

File: _data/icons/edexa.json (hunk 2 above)
Comment (ranjith-edx) [reply _data/icons/edexa.json]: updated as png @ligi

**Review by ligi — APPROVED** [review]

File: _data/icons/edexa.json (hunk 2 above)
Comment (ranjith-edx) [reply _data/icons/edexa.json]: Hi @ligi  which link will it get reflected i'm not able to see the chain in website
